Validator/Validator.cs

```diff
@@ -16,7 +16,10 @@ public bool ValidateCategory(string category)
 
     public bool ValidateDeadline(DateTime deadline)
     {
-        // Add validation logic here
+        if(deadline < DateTime.Now)
+        {
+            return false;
+        }
         return true;
     }
```

Comment:
Måske tilføj et upper-limit også, så den ikke kan gå for langt ud i fremtiden, eller hvad tænker du?

Reply:
jeg kan godt hvis du vil have det. Men er det must have?